src/main/java/com/gregtechceu/gtceu/api/item/component/ElectricStats.java

```diff
@@ -184,9 +184,9 @@ private static void setInDischargeMode(ItemStack itemStack, boolean isDischargeM
     public void appendHoverText(ItemStack stack, @Nullable Level level, List<Component> tooltipComponents,
                                 TooltipFlag isAdvanced) {
         IElectricItem electricItem = GTCapabilityHelper.getElectricItem(stack);
-        if (electricItem != null && electricItem.canProvideChargeExternally()) {
+        if (electricItem != null) {
             addCurrentChargeTooltip(tooltipComponents, electricItem.getCharge(), electricItem.getMaxCharge(),
-                    electricItem.getTier(), true);
+                    electricItem.getTier(), electricItem.canProvideChargeExternally());
```

Comment:
you can do it like this:
```suggestion
        if (electricItem == null) return;
        addCurrentChargeTooltip(tooltipComponents, electricItem.getCharge(), electricItem.getMaxCharge(),
                electricItem.getTier(), electricItem.canProvideChargeExternally());
        if (electricItem.canProvideChargeExternally()) {
```